### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by harrisaoz**
#### Summary
Fix a common its vs it's mistake.

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/css/filter/index.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ When animated, if both the beginning and end filters have a function list of the
 
 The `filter` property is specified as `none` or one or more of the functions listed below. If the parameter for any function is invalid, the function returns `none`. Except where noted, the functions that take a value expressed with a percent sign (as in `34%`) also accept the value expressed as decimal (as in `0.34`).
 
-When a single `filter` property has two or more functions it's results will be different from when two or more `filter` properties are separately applied with the same functions.
+When a single `filter` property has two or more functions its results will be different from when two or more `filter` properties are separately applied with the same functions.
```
Comment (OnkarRuikar) [inline files/en-us/web/css/filter/index.md:73]:
```suggestion
When a `filter` property has two or more functions, its results are different from the same functions applied separately using multiple `filter` properties.
```
- 'single' is implied when 'a' is used
- The original statement doesn't properly convey that we want to apply each function separately.

**Review by teoli2003 — APPROVED** [review]
I went ahead and committed Onkar's suggestion.

👍

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! :tada:

**Comment by harrisaoz** [comment]
Thanks. The project seems to be in very capable hands! I’m impressed with
the responsiveness of the team.

**Pull request merged**